sanic/utils.py

```diff
@@ -0,0 +1,82 @@
+from os import environ as os_environ
+from re import findall as re_findall
+from importlib.util import spec_from_file_location, module_from_spec
+from .exceptions import LoadFileException
+from typing import Union
+
+
+def str_to_bool(val):
+    """Takes string and tries to turn it into bool as human would do.  
+
+    If val is in case insensitive ("y", "yes", "yep", "yup", "t", "true", "on", "enable", "enabled", "1") returns True.  
+    If val is in case insensitive ("n", "no", "f", "false", "off", "disable", "disabled", "0") returns False.  
+    Else Raise ValueError."""
+
+    val = val.lower()
+    if val in {
+        "y",
+        "yes",
+        "yep",
+        "yup",
+        "t",
+        "true",
+        "on",
+        "enable",
+        "enabled",
+        "1",
+    }:
+        return True
+    elif val in {"n", "no", "f", "false", "off", "disable", "disabled", "0"}:
```

Comment:
make fix-imports  
formated only "true" set, but not "false" set,  
Not quite ok ? Should do the same for both ? Or am I wrong ?

Reply:
Probably has to do with the width. I would not worry about it.

Reply:
OK.